helpers/deploymentConfig.ts

```diff
@@ -179,6 +179,24 @@ export const getGlobalConfig: () => Promise<NetworkConfig> = async () => {
           decimals: 18,
           tokenAddress: "0xe73774DfCD551BF75650772dC2cC56a2B6323453",
         },
+        {
+          isMock: true,
+          name: "PT Ethena sUSDE 26JUN2025",
```

Comment:
```suggestion
          name: "PT Ethena sUSDE 27MAR2025",
```
As mentioned on ticket?